client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -11,6 +18,17 @@ const Notices = ( {
 		noticeOperations.removeNotice( id );
```

Comment:
It would be better to remove the timer the notice is going to be removed.

Reply:
I pushed a change. How does it look?